portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +15,18 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    ArrayList<String> greetings = new ArrayList<>();
+    greetings.add("hello");
+    greetings.add("hi");
+    greetings.add("whats up");
```

Comment:
Not sure on the exact right syntax, but does 
 ArrayList<String> greetings = new ArrayList<>("hello", "hi", "whats up");
work?